<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Create new user</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>First name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtFirstName"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Last name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtLastName"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="CountryCombo" name="comboCountry"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Province</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtProvince"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtCity"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Zip Code</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtZipCode"/>
          </item>
         </layout>
        </item>
        <item row="6" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Birthdate</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="CurrentDateEdit" name="dateBirthdate"/>
          </item>
         </layout>
        </item>
        <item row="7" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Phone number</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtPhoneNumber"/>
          </item>
         </layout>
        </item>
        <item row="8" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_9">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="EmailTextInput" name="txtEmail"/>
          </item>
         </layout>
        </item>
        <item row="9" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="UsernameTextInput" name="txtUsername"/>
          </item>
         </layout>
        </item>
        <item row="10" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_11">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PasswordTextInput" name="txtPassword"/>
          </item>
         </layout>
        </item>
        <item row="11" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_12">
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Confirm password</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PasswordTextInput" name="txtConfirmPassword"/>
          </item>
         </layout>
        </item>
        <item row="12" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_13">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>LinkedIn Profile URL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="UrlTextInput" name="txtLinkedInProfileUrl"/>
          </item>
         </layout>
        </item>
        <item row="13" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>808</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="13" column="1">
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>125</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Edit user</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="5">
         <widget class="UserTableWidget" name="tableWidget"/>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>808</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="editButton">
          <property name="text">
           <string>Edit (OFF)</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/img/unlocked.png</normaloff>:/img/unlocked.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="restoreButton">
          <property name="text">
           <string>Restore deleted items</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="saveEditionButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CountryCombo</class>
   <extends>QComboBox</extends>
   <header location="global">shared/widgets/countrycombo.h</header>
  </customwidget>
  <customwidget>
   <class>EmailTextInput</class>
   <extends>QLineEdit</extends>
   <header location="global">shared/widgets/emailtextinput.h</header>
  </customwidget>
  <customwidget>
   <class>UsernameTextInput</class>
   <extends>QLineEdit</extends>
   <header location="global">shared/widgets/usernametextinput.h</header>
  </customwidget>
  <customwidget>
   <class>PasswordTextInput</class>
   <extends>QLineEdit</extends>
   <header location="global">shared/widgets/passwordtextinput.h</header>
  </customwidget>
  <customwidget>
   <class>UrlTextInput</class>
   <extends>QLineEdit</extends>
   <header location="global">shared/widgets/urltextinput.h</header>
  </customwidget>
  <customwidget>
   <class>CurrentDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">shared/widgets/currentdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>UserTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">shared/widgets/usertablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
